source/MaterialXView/Editor.cpp

```diff
@@ -458,7 +458,6 @@ void PropertyEditor::addItemToForm(const mx::UIPropertyItem& item, const std::st
             }
         });
         v1->setSpinnable(editable);
-        v1->setEditable(editable);
```

Comment:
These should not be removed (unless it's set somewhere else now). If we don't set the control to editable we can't type in values in the control, only spin the spinner, which sometimes is not high precision enough if you need more decimals.

Reply:
(same for all below)